app/controllers/api/v1x0/workflows_controller.rb

```diff
@@ -1,23 +1,20 @@
 module Api
   module V1x0
     class WorkflowsController < ApplicationController
-      include Api::V1x0::Mixins::IndexMixin
-      include Api::V1x0::Mixins::RBACMixin
-
-      before_action :create_access_check, :only => %i[create]
-      before_action :update_access_check, :only => %i[update]
-      before_action :destroy_access_check, :only => %i[destroy]
+      include Mixins::IndexMixin
 
       def create
+        authorize Workflow
+
         workflow = WorkflowCreateService.new(params.require(:template_id)).create(params_for_create)
         json_response(workflow, :created)
       end
 
-      # TODO: remove 'approval:workflows:read' from approver acls list in RBAC Insight
       def show
-        raise Exceptions::NotAuthorizedError, "Not Authorized for workflows" if Insights::API::Common::RBAC::Access.enabled? && !admin?
```

Comment:
@hsong-rh This is a good change, are you removing all RBAC enabled check and assuming every spec is going to be rbac enabled like it should be.

Reply:
I moved the check to https://github.com/RedHatInsights/approval-api/blob/a1806ddd501e19ed2dca133799dd1092f826c38a/app/policies/mixins/rbac_mixin.rb#L14. It will be always called there.